Base: Improve docstrings in BoundBoxPy.xml

marioalexis84 opened the pull request:
Thank you for creating a pull request to contribute to FreeCAD! To ease integration, we ask you to conform to the following items. Pull requests which don't satisfy all the items below might be rejected. If you are in doubt with any of the items below, don't hesitate to ask for help in the [FreeCAD forum](https://forum.freecadweb.org/viewforum.php?f=10)!

- [ ]  Your pull request is confined strictly to a single module. That is, all the files changed by your pull request are either in `App`, `Base`, `Gui` or one of the `Mod` subfolders. If you need to make changes in several locations, make several pull requests and wait for the first one to be merged before submitting the next ones
- [ ]  In case your pull request does more than just fixing small bugs, make sure you discussed your ideas with other developers on the FreeCAD forum
- [ ]  Your branch is [rebased](https://git-scm.com/docs/git-rebase) on latest master `git pull --rebase upstream master`
- [ ]  All FreeCAD unit tests are confirmed to pass by running `./bin/FreeCAD --run-test 0`
- [ ]  All commit messages are [well-written](https://chris.beams.io/posts/git-commit/) ex: `Fixes typo in Draft Move command text`
- [ ]  Your pull request is well written and has a good description, and its title starts with the module name, ex: `Draft: Fixed typos`
- [ ]  Commit messages include `issue #<id>` or `fixes #<id>` where `<id>` is the issue ID number from our [Issues database](https://github.com/FreeCAD/FreeCAD/issues) in case a particular commit solves or is related to an existing issue. Ex: `Draft: fix typos - fixes #4805`

And please remember to update the Wiki with the features added or changed once this PR is merged.  
**Note**: If you don't have wiki access, then please mention your contribution on the [0.20 Changelog Forum Thread](https://forum.freecadweb.org/viewtopic.php?f=10&t=56135).

---

freecadci commented:
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7012"><img alt="pipeline status" src="https://gitlab.com/freecad/FreeCAD-CI/badges/PR_7012/pipeline.svg"/></a> for feature branch [PR_7012](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7012). Pipeline [561809527](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/561809527) was triggered at [429692b](https://github.com/FreeCAD/FreeCAD/pull/7012/commits/429692b7505c6a36a33c3932f58be071b44fb5ba). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

donovaly commented:
@chennes , could you review please this and merge if OK?

chennes commented:
@marioalexis84 I notice that you removed the incorrectly-formatted error string entirely. Is there a reason you decided not to replace the `:` with a `;` and leave the error message intact?

marioalexis84 commented:
I removed the string error entirely to use the default error message since it already indicates when the data type is wrong.
But I guess the original post can be considered more explanatory, so I've reverted it.
Also removed other unused error messages and edit the `if` block  in `isCutPlane` function and the error string.

freecadci commented:
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7012"><img alt="pipeline status" src="https://gitlab.com/freecad/FreeCAD-CI/badges/PR_7012/pipeline.svg"/></a> for feature branch [PR_7012](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7012). Pipeline [561967295](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/561967295) was triggered at [9ce8f3e](https://github.com/FreeCAD/FreeCAD/pull/7012/commits/9ce8f3ecaca1e4b131f91cf67a7559ad7bcdd4d8). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).
